CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ Release Notes
 
 * StrLiteral is no longer converted to CHAR(<length>)
 
-fix(isthmus): convert BinaryLiteral to VARBINARY 
+fix(isthmus): convert BinaryLiteral to VARBINARY
```

Comment:
nit: maybe revert whitespace change?

Reply:
I got a formatting issue, that's why I added it, but let me check it again.

Reply:
@danepitkin I just went through NLJ PR to Substrait-Java, and I see something similar.

https://github.com/substrait-io/substrait-java/actions/runs/6475823135/job/17583520911#step:4:1

Reply:
ohhh, is that what my error is?? Good catch, thank you

Reply:
It seems like that, but how could it be there in the first place?

Reply:
This is extra weird because the changelog is generated programatically 🤔 

Ah, the whitespace is in the commit message itself
https://github.com/substrait-io/substrait-java/commit/6e82f397a168f3811593fe8248753d70d4f0c04f

I think it's fine to fix it like this.